### DISCUSSION TITLE
Fix Japanese input (mode without spaces)

### DISCUSSION THREAD

**Discussion by SamCyanide** [monkeytypegame/monkeytype#5039] [Ideas]
When writing Japanese, spaces between words are generally not used. They are only used for after a 。 at the end of a sentence, and the character itself includes the space. The tests on MonkeyType currently have spaces between words, so you have to do at least 3 additional inputs if you are using an IME. This slows down typing to a pace much slower than typing actual Japanese. 

The spaces should be removed or a new mode should be added to emulate the way Japanese is written.

<img width="1085" alt="image" src="https://github.com/monkeytypegame/monkeytype/assets/9272499/77e35fef-ca16-47d7-b621-837b2a3cc884">

vs a snippet from NHK News:
<img width="716" alt="image" src="https://github.com/monkeytypegame/monkeytype/assets/9272499/87f05e6a-efd3-4277-bf8d-511735794f41">

**Comment by Miodec** [comment]
Have you tried using the `nospace` funbox?

**Reply by SamCyanide** [reply]
I wasn't aware of this - thanks for the suggestion. Trying this mode did make me realize though, having spaces (visually) might actually not be so bad. Without Kanji its difficult to find the ends of words so it slows me down as I try to process those ends. 

One solution might be to keep the spaces visually, but not require an actual spacebar input. This would help with the differentiation of each word without requiring an input that you usually don't need to do when typing hiragana.

**Reply by enkvadrat** [reply]
With nospace funbox the result will not count to your pb

**Comment by lethargistic** [comment]
would want some half solution for this like skipping spaces as well
